astropy/visualization/units.py

```diff
@@ -48,7 +50,7 @@ def rad_fn(x, pos=None):
         else:
             return f"{n}π/2"
 
-    class MplQuantityConverter(units.ConversionInterface):
+    class MplQuantityConverter(ContextDecorator, units.ConversionInterface):
```

Comment:
I'm thinking `ContextDecorator` should probably be the second base class rather than the first, so `units.ConversionInterface` remains the first place that's looked up for attributes and methods.

That is also how the class is used in a similar example from the documentation
https://docs.python.org/3/library/contextlib.html#contextlib.ContextDecorator

Reply:
Done.